src/util/alias/get-deployment-from-alias.ts

```diff
@@ -9,7 +8,7 @@ export default async function fetchDeploymentFromAlias(
   prevAlias: Alias | null,
   currentDeployment: Deployment
 ) {
-  return prevAlias && prevAlias.deploymentId !== currentDeployment.uid
```

Comment:
Is this check needed?

Reply:
At least there needs to be a check if `deploymentId` exists. The rest I'm not sure about, you can decide that after this PR has been shipped though.

Reply:
Wrote it in the wrong place. I meant @timneutkens that:

```
return prevAlias && prevAlias.deploymentId && prevAlias.deploymentId !== currentDeployment.uid
```

What's the value of checking `prevAlias.deploymentId`? Is it because when both that and `currentDeployment.uid` it would be. false positive?

Reply:
@javivelasco `prevAlias.deploymentId` can be undefined because the previous deployment could be a rules.json alias which does not have `deploymentId` as a field.

Reply:
I read `===` 👍